test/spec/directive.coffee

```diff
@@ -77,6 +77,22 @@ describe 'angular-masonry', ->
     expect(@scope.$watch).not.toHaveBeenCalled()
   )
 
+  it 'should setup a $watch when the reload-on-resize is present', inject(($compile) =>
+    sinon.spy(@scope, '$watch')
+    element = angular.element '<masonry reload-on-resize></masonry>'
+    element = $compile(element)(@scope)
+
+    expect(@scope.$watch).toHaveBeenCalled()
```

Comment:
I based this test on a previous one (1b6e2b0ba41dbe294136987d925c35b6f480a2d9). I understand the problem, but I can't think of any way to solve it.